src/clj/test/cards/events.clj

```diff
@@ -644,6 +644,76 @@
     (is (= 0 (:credit (get-runner))))
     (is (= 3 (count (:discard (get-runner)))))))
 
+(deftest falsified-credentials
+  ;; Falsified Credentials - Expose card in remote
+  ;; server and correctly guess its type to gain 5 creds
+  (do-game
+    (new-game (default-corp [(qty "Eve Campaign" 2)
+                             (qty "Product Placement" 2)
+                             (qty "Project Atlas" 1)])
+              (default-runner [(qty "Falsified Credentials" 5)]))
```

Comment:
I should've mentioned this on your other tests, and we'll fix those eventually, but it's probably best if we stick to deckbuilding rules as much as possible, particularly re: maximum number of copies. There's no deck validation in the test suite, but just in case...

Reply:
Point well taken, but I don't think I have any other tests. Unless I'm forgetting some maybe you're thinking of someone else.